src/tests/ftest/cart/util/cart_utils.py

```diff
@@ -381,6 +389,37 @@ def build_cmd(self, env, host, **kwargs):
 
         return tst_cmd
 
+    def convert_xml(self, xml_file):
+        """Modify the xml file"""
+
+        fd = open(xml_file, 'r')
```

Comment:
(pylint-consider-using-with) Consider using 'with' for resource-allocating operations